### ISSUE TITLE
nbtutor + nbgrader create_assignment extensions = broken notebook

### ISSUE THREAD

**Issue by lgpage** [lgpage/nbtutor#30]
``nbtutor`` makes notebook un-usable when both ``nbtutor`` and ``nbgrader`` ``create_assignment`` extensions are enabled. It is ``nbtutor`` that is breaking things with the following web console error:
```
too much recursion
  initEvents/CellToolbar.prototype.rebuild       nbtutor/js/nbtutor.min.js:48:1
```

**Comment by sigurdurb** [comment]
This seems to be an issue still, I am running this with jupyterhub0.8.1, anaconda with python3.6, ,notebook5.0.4, nbgrader0.6.0dev and nbtutor1.0.4.  But for me the issue was actually in the rebuild method of the nbextensions/create_assignment/main.js 
or at least that was what firefox on my Ubuntu complained about. 
This works however if I disable the extension and only have the nbgrader validate assignment running. I tried removing some code and debuggin but I have no idea how these could be linked.
Should we open an issue on this also on the nbgrader repo ?

**Comment by psychemedia** [comment]
I've just encountered this issue too running a recent version of `nbgrader` and Jupyter notebook.

**Comment by lgpage** [comment]
Should be fixed by #44
